recipe/meta.yaml

```diff
@@ -27,7 +27,7 @@ outputs:
         - libparallelproj =*=cuda*  # [cuda_compiler_version != "None"]
         - libparallelproj =*=cpu*  # [cuda_compiler_version == "None"]
       ignore_run_exports:
-        - cudatoolkit
+        - cudatoolkit  # [cuda_compiler_version == "11.*"]
```

Comment:
```suggestion
        - cudatoolkit  # [(cuda_compiler_version or "None").startswith("11")]
```

Reply:
many thanks. your help is much appreciated. added your suggestion in a this new PR https://github.com/conda-forge/parallelproj-feedstock/pull/77